learning_wavelets/models/learnlet_layers.py

```diff
@@ -151,9 +166,10 @@ def __init__(
                 name=f'{tiling_prefix}_{str(K.get_uid(tiling_prefix))}',
             ) for i in range(self.n_scales)
         ]
+
         if self.mixing_details:
             mixing_prefix = 'details_mixing'
-            self.convs_detail_mixing = [
+            self.convs_detail_mixing_fixed = [
```

Comment:
```suggestion
            self.convs_detail_mixing = [
```

I think this part doesn't need to be changed

Reply:
right